### PR TITLE
consistent & improved engine descriptions

### PR DESCRIPTION
**Pull request by wfleming**
* always end with a period
* some descriptions were just the engine name: I've tried to write
  better descriptions for these.

cc @codeclimate/review

### CODE REVIEW

File: config/engines.yml [hunk 1]
```diff
@@ -94,14 +94,14 @@ govet:
     - "**.go"
 fixme:
   image: codeclimate/codeclimate-fixme
-  description: Find FIXME, TODO, HACK, etc. comments
+  description: Find FIXME, TODO, HACK, etc. comments.
```
Comment (mxie) [inline config/engines.yml]: `Finds`

File: config/engines.yml [hunk 2]
```diff
@@ -30,7 +30,7 @@ bundler-audit:
     - Gemfile.lock
 csslint:
   image: codeclimate/codeclimate-csslint
-  description: Automated linting of Cascading Stylesheets
+  description: Automated linting of Cascading Stylesheets.
```
Comment (mxie) [inline config/engines.yml]: Could we just say `CSS`?

**Comment by GordonDiggs** [comment]
:+1:

**Comment by mxie** [comment]
A few nits. LGTM otherwise.

File: config/engines.yml (hunk 2 above)
Comment (wfleming) [inline config/engines.yml]: This comes pretty much directly from the project page, and we try to copy/imitate official descriptions where possible.

Comment (mxie) [inline config/engines.yml]: Ah ok. Sure!

**Pull request merged**